mmv1/products/securitycenterv2/FolderMuteConfig.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'FolderMuteConfig'
+base_url: '{{parent}}/foldermuteConfigs'
+self_link: '{{name}}'
+create_url: '{{parent}}/foldermuteConfigs?foldermuteConfigId={{folder_mute_config_id}}'
```

Comment:
This is a different query parameter than is listed here: https://cloud.google.com/security-command-center/docs/reference/rest/v2/folders.locations.muteConfigs/create

Reply:
addressed